frontend/index.html

```diff
@@ -17,46 +17,54 @@
+                <div class="search-icons container"><img class="searchicons" src="./assets/images/geo-alt.svg" alt="geo locator">
+                    <span>Stores</span>
+                    </div>
+            </a>
+            <div class="logo">
                <p>DIYToolWorld</p>
             </div>
-            <div class="menu__item">
+            <a class="nav-icons" href="#ideas">
+                <div class="search-icons container"><img  class="searchicons" src="./assets/images/lightbulb.svg" alt="lightbulb" >
+                    <span>Ideas & Advice</span></div>
+            </a>
+            
+            <a class="nav-icons" href="#basket">
+                <div class="search-icons container"> <img img class="searchicons" src="./assets/images/basket.svg" alt="basket">
+                    <span>Basket</span>
+                </div>  
+            </a>
+            <div class="search-container">
                 <input class="search-bar" type="search" placeholder="Search...">
-            </div>
-             <div class="menu__item">
-                <div class="icons-list">
-                    <div class="ideas">
-                        <a href="#ideas">
-                            <div><img  class="searchicons" src="./assets/images/lightbulb.svg" alt="" >
-                                <p>Ideas & Advice</p></div>
-                        </a> 
-                    </div>
-                    <div class="ideas">
-                        <a href="#stores">
-                            <div><img class="searchicons" src="./assets/images/geo-alt.svg" alt="">
-                                <p>Stores</p></div>
-                        </a>
-                        </div>
-                        <div class="ideas">
-                            <a href="#basket">
-                                <div> <img img class="searchicons" src="./assets/images/basket.svg" alt="">
-                                    <p>Basket</p>
-                                </div>
-                            </a>
-                        </div>
-                    </div>
-                </div>
-             </div>
+            </div> 
+        </div>                            
     </header>
 
-        <nav class="categories container nav-color">
-            <ul class="categories-list">
+        <nav class="categories container nav-color ">
+            <ul class="sidebar">
+                <li onclick=hideSideBar()><a href="#close"><img class="close" src="./assets/images/x.svg" alt=""></a></li>
                 <li><a href="#decorations">Decorations</a></li>
                 <li><a href="#garden">Garden</a></li>
                 <li><a href="#kitchen">Kitchen</a></li>
                 <li><a href="#tools">Tools</a></li>
                 <li><a href="#storage">Storage</a></li>
             </ul>
+            <ul class="categories-list">
+                <li class="hideOnMobile"><a href="#decorations">Decorations</a></li>
```

Comment:
Is the reason for only hiding the list items (and not the entire list itself) to keep the space that the list takes up in mobile as well? 

I would probably prefer using something like 
```
categories-list > li {
   @media (max-width: 1180px) {
      display: none;
   }
}
```

in that case. 

Alternatively, if we hide the entire list, could we achieve the spacing in another way?

Reply:
Reset css code has been added in and a margin-bottom of 1.5rem has been applied to the header